<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TC1ANB6" failures="2" tests="5" name="Test" time="0.248" errors="0" timestamp="2025-06-01T19:50:09 IST">
  <testcase classname="classPracticeSession.ListnerLoginPageTC" name="testcase1" time="0.065">
    <failure type="java.lang.AssertionError" message="expected [4] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [4] but found [2]
at classPracticeSession.ListnerLoginPageTC.testcase1(ListnerLoginPageTC.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testcase1 -->
  <testcase classname="classPracticeSession.ListnerLoginPageTC" name="testcase2" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at classPracticeSession.ListnerLoginPageTC.testcase2(ListnerLoginPageTC.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testcase2 -->
  <testcase classname="classPracticeSession.ListnerLoginPageTC" name="testcase3" time="0.001"/>
  <testcase classname="classPracticeSession.ListnerLoginPageTC" name="testcase4" time="0.009"/>
  <testcase classname="classPracticeSession.ListnerLoginPageTC" name="testcase5" time="0.004">
    <skipped/>
  </testcase> <!-- testcase5 -->
</testsuite> <!-- Test -->
